setuptools/independent/DepsProcessor/project.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "NuGet.RuntimeModel": "3.5.0-rc-1285",
     "NuGet.Versioning": "3.5.0-rc-1285",
-    "System.CommandLine": "0.1.0-e160119-1",
+    "System.CommandLine": "0.1.0-e170322-6",
```

Comment:
If this is the reason we needed the corefxlab feed then we should either downgrade or move this package to the dotnet-core feed instead.